app/controllers/curriculum-inventory-report/index.js

```diff
@@ -1,11 +1,31 @@
 import Controller from '@ember/controller';
 import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
 import { use } from 'ember-could-get-used-to-this';
 import ResolveAsyncValue from 'ilios-common/classes/resolve-async-value';
 
 export default class CurriculumInventoryReportIndexController extends Controller {
+  queryParams = ['leadershipDetails', 'manageLeadership'];
+  @tracked hasUpdatePermissions = false;
+  @tracked leadershipDetails = false;
+  @tracked manageLeadership = false;
+  @tracked isFinalized = true;
```

Comment:
I'm not seeing how this value gets set initially?

Reply:
Ah ok, it's based on `!!model.belongsTo('export').id()` in the route. Instead of this going in the route it could just be done here when the variable is initialized.

```suggestion
  @tracked isFinalized = this.model.belongsTo('export').id();
```

Reply:
:heavy_check_mark: